### DISCUSSION TITLE
Are there unknown achievements in GitHub

### DISCUSSION THREAD

**Discussion by andriymelnyk1997** [community/community#37089] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi everyone.

Lately, I met this screenshot in the official docs of GitHub, and I am wondering: are there new achievements that have not been published yet, or are the conditions to achieve unknown?
<img width="349" alt="achievements-on-profile" src="https://user-images.githubusercontent.com/46633744/197480303-e3d7dcf9-767a-4404-ba87-aec30417f77d.png">

The source document link: https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/showing-your-private-contributions-and-achievements-on-your-profile

Any thoughts?

**Comment by godnondsilva** [comment]
Interesting, there is literally no information about these achievements whatsoever.

The best guess I can take is that these achievements have been updated now.
Like the "Heart on your sleeve" could be the "Public sponsor" and the  "Open source....(not sure)" could be "Starstruck". Or the opposite if you consider the sub achievements (3x on Heart on your sleeve)

**Reply by andriymelnyk1997** [reply]
Very possible version! Did not think that way :) Thanks, @godnondsilva, for your time.

**Comment by melvincwng** [comment]
@OpenSourceLucy Hi Lucy, those are probably new achievements that have yet to be released/published. Hence we do not currently know what conditions are needed to satisfy those achievements.

That screenshot might be a little teaser by the GitHub team to showcase what future achievements could be unlocked by us in future.

Above is strictly just my opinion on this matter, hope this clarifies 😊

**Comment by ossanna16** [comment]
Hi there @OpenSourceLucy and welcome to our community! Thank you for asking a great question 🙂

**Comment by ArtyomVancyan** [comment, marked as answer]
The existence of **Heart On Your Sleeve** and **Open Sourcerer** achievements were publicly verified a few days ago at: https://github.com/Schweinepriester/github-profile-achievements#achievements, and soon, we will know their preconditions and earnability.

**Reply by andriymelnyk1997** [reply]
Oh, thanks. This was very helpful information. Now we know that there are new achievements, though.

**Comment by github-staff** [comment]
Thanks so much for posting in our Community @OpenSourceLucy. Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀

**Comment by ghost** [comment]
It seems that earning these badges has become harder, perhaps because GitHub is testing how they can be achieved. However, I haven’t figured out the exact criteria for earning these badges yet.

**Comment by dnnr1** [comment]
lol yesterday these achievements appeared on my profile for a few moments and then disappeared, amazing!
